pandas/_libs/reduction.pyx

```diff
@@ -17,6 +17,7 @@ cnp.import_array()
 
 cimport util
 from lib import maybe_convert_objects
+from tslibs.conversion import NS_DTYPE, TD_DTYPE
```

Comment:
are these imports used?

Reply:
They were unused. Should that be a lint error?

Reply:
IIUC correctly we don't include error code F401 as part of the list for Cython LINTing. Maybe should though

Reply:
Yah, I periodically do a manual check for unused imports in the cython files.